File: src/main/java/com/example/project_test/auth/RegisterRequest.java
package com.example.project_test.auth;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class RegisterRequest {
    private String nomEt;
    private String prenomEt;
    private String email;
    private String password;
}

File: src/app/model/register-request.ts
export interface RegisterRequest{
  nomEt?:string;
  prenomEt?:string;
  email?:string;
  password?:string;
  role?:string;
  cin?: number;
  ecole?: string;
  dateNaissance?: Date;
  mfaEnabled?:string;

}
